libmscore/score.h

```diff
@@ -376,6 +376,7 @@ class Score : public QObject, public ScoreElement {
       QString _layerTagComments[32];
       QList<Layer> _layer;
       int _currentLayer { 0 };
+      int _isAnnotation { 0 };
```

Comment:
I don't see a reason why this shouldn't be done the same as all other similar tester functions - see element.h, the CONVERT macros.